website/docs/foundations/icons/partials/engineering-guidelines.md

```diff
@@ -34,19 +34,24 @@ To install, run:
 yarn add @hashicorp/ember-flight-icons
 ```
 
-Note: Because this addon exposes a `data-test-icon` helper it is suggested consumers install `ember-test-selectors`. [This Ember addon](https://github.com/simplabs/ember-test-selectors) strips out all `data-test-*` attributes for production builds.
+!!! Information
+
+**Note**
+
+Because this addon exposes a `data-test-icon` helper it is suggested consumers install `ember-test-selectors`. [This Ember addon](https://github.com/simplabs/ember-test-selectors) strips out all `data-test-*` attributes for production builds.
```

Comment:
```suggestion
Because this addon exposes a `data-test-icon` helper, it is suggested consumers install `ember-test-selectors`. [This Ember addon](https://github.com/simplabs/ember-test-selectors) strips out all `data-test-*` attributes for production builds.
```